CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedIO.java

```diff
@@ -230,6 +231,14 @@ private static void loadParameterFile(final DelimitedImportController importCont
             final JsonNode source = root.get(SOURCE);
             final String parser = source.get(PARSER).textValue();
             final ImportFileParser ifp = ImportFileParser.getParser(parser);
+
+            ImportFileParser importFileParser = importController.getImportFileParser();
+
+            if (!importController.getImportFileParser().getLabel().equals(parser)) {
```

Comment:
I note that you define a variable for the file parser but then don't use it in the subsequent line. Not sure if that was accidentally missed?

Reply:
Removed it.